packages/youtube_player_iframe/lib/src/helpers/player_fragments.dart

```diff
@@ -36,7 +36,7 @@ String youtubeIFrameTag(YoutubePlayerController controller) {
   return '<iframe id="player" type="text/html"'
       ' style="position:absolute; top:0px; left:0px; bottom:0px; right:10px;'
       ' width:100%; height:100%; border:none; margin:0; padding:0; overflow:hidden; z-index:999999;"'
-      ' src="$sourceUri" frameborder="0" allowfullscreen></iframe>';
+      ' src="$sourceUri" frameborder="0" allow="autoplay;fullscreen" allowfullscreen></iframe>';
```

Comment:
We should be able to remove `allowfullscreen`.

Reply:
@supermuesli @sarbagyastha Can  you please merge this changes it its works